Makefile

```diff
@@ -2,7 +2,7 @@ install:
 	pip install -e .[dev]
 
 debug:
-	FLASK_APP=policyengine_api.api FLASK_DEBUG=1 flask run --without-threads
+	FLASK_APP=policyengine_api.api FLASK_DEBUG=1 flask run --without-threads -p 5050
```

Comment:
Not sure I understand this port? But not a big issue

Reply:
Sorry, that was an error introduced when debugging locally, as 5000 was occupied; I could open an issue to remove that